content/hi/docs/contribute/style/page-content-types.md

```diff
@@ -0,0 +1,215 @@
+---
+title: पृष्ठ सामग्री प्रकार
+content_type: concept
+weight: 30
+card:
+  name: contribute
+  weight: 30
+---
+
+<!-- overview -->
+
+कुबेरनेट्स प्रलेखन कई प्रकार की पृष्ठ सामग्री का अनुसरण करता है:
+
+- संकल्पना
+- कार्य
+- ट्यूटोरियल
+- संदर्भ
+
+<!-- body -->
+
+## सामग्री अनुभाग {#content-sections}
+
+प्रत्येक पृष्ठ सामग्री प्रकार में Markdown टिप्पणियों और HTML शीर्षकों 
+द्वारा परिभाषित कई खंड होते हैं। आप `heading` शॉर्टकोड का उपयोग करके अपने पृष्ठों
+में विषय शीर्षक जोड़ सकते हैं। टिप्पणियाँ और शीर्षक पृष्ठ सामग्री प्रकारों की संरचना 
+को बनाए रखने में मदद करते हैं।
+
+पृष्ठ सामग्री अनुभागों को परिभाषित करने वाली Markdown टिप्पणियों के उदाहरण:
+
+```markdown
+<!-- overview -->
+```
+
+```markdown
+<!-- body -->
+```
+
+अपने सामग्री पृष्ठों में सामान्य शीर्षक बनाने के लिए, `heading` शोर्टकोड का उपयोग करें
+एक शीर्षक स्ट्रिंग।
+
+हेडिंग स्ट्रिंग्स के उदाहरण:
+
+- whatsnext
+- prerequisites
+- objectives
+- cleanup
+- synopsis
+- seealso
+- options
+
+उदाहरण के लिए, एक `Whatsnext` शीर्षक बनाने के लिए, "whatsnext" स्ट्रिंग के साथ शीर्षक शोर्टकोड जोड़ें:
+
+```none
+## {{%/* heading "whatsnext" */%}}
+```
+
+आप एक `prerequisites` शीर्षक इस प्रकार घोषित कर सकते हैं:
+
+```none
+## {{%/* heading "prerequisites" */%}}
+```
+
+`heading` शॉर्टकोड एक स्ट्रिंग पैरामीटर की अपेक्षा करता है।
+शीर्षक स्ट्रिंग पैरामीटर `i18n/<lang>.toml` फ़ाइलों में एक वैरिएबल के उपसर्ग से मेल खाता है।
+उदाहरण के लिए:
+
+`i18n/en.toml`:
+
+```toml
+[whatsnext_heading]
+other = "What's next"
+```
+
+`i18n/ko.toml`:
+
+```toml
+[whatsnext_heading]
+other = "다음 내용"
+```
+
+## सामग्री प्रकार {#content-types}
+
+प्रत्येक सामग्री प्रकार अनौपचारिक रूप से इसकी अपेक्षित पेज संरचना को परिभाषित करता है।
+सुझाए गए पृष्ठ अनुभागों के साथ पृष्ठ सामग्री बनाएँ।
+
+### संकल्पना {#concept}
+
+एक अवधारणा पृष्ठ कुबेरनेट्स के कुछ पहलू की व्याख्या करता है। उदाहरण के लिए, एक अवधारणा
+पृष्ठ कुबेरनेट्स डिप्लॉयमेंट ऑब्जेक्ट का वर्णन कर सकता है एक बार इसे तैनात, स्केल और अपडेट करने के बाद
+एक एप्लिकेशन के रूप में इसकी भूमिका की व्याख्या कर सकता है। आमतौर पर, अवधारणा
+पृष्ठ में चरणों का क्रम शामिल नहीं है, बल्कि कार्यों या ट्यूटोरियल के लिंक प्रदान करता है।
+
+एक नया अवधारणा पृष्ठ लिखने के लिए, `/content/en/docs/concepts` निर्देशिका की एक उपनिर्देशिका में
+एक Markdown फ़ाइल बनाएँ, निम्नलिखित विशेषताओं के साथ:
+
+अवधारणा पेज तीन खंडों में विभाजित हैं:
+
+| Page section  |
```

Comment:
Entire table should be localized since we are explaining what each concept page comprises of.